ChangeLog

```diff
@@ -113,7 +113,10 @@
  * #2025 (The lack of Bonmin generates an error during example compilation)
  * #2031 (Multi-objective optimization example question)
  * #2032 (Documentation of NAIS)
-
+ * #2097 (LinearModelStepwiseAlgorithm ctor order)
+ * #2121 (GaussianProcess Gibbs sampling method issues)
+ * #2123 (MixtureClassifier 'grade' method does not work with a Sample input)
+ * #2125 (SciPyDistribution __init__ failure with scipy 1.9.0)
```

Comment:
this is the previous 1.19 changelog
I suggest to simply drop these as we agreed to stop editing the bugfixes paragraph to avoid conflicts

Reply:
I moved this to 1.20 section as there is alrerady an empty section